src/test/java/com/google/api/codegen/testdata/no_path_templates_config.baseline

```diff
@@ -36,10 +54,20 @@ interfaces:
     rpc_timeout_multiplier: 1
     max_rpc_timeout_millis: 20000
     total_timeout_millis: 600000
+  # A list of method configurations.
   methods:
+    # The simple name of the method.
   - name: Increment
+    # Turns on or off the generation of a method whose sole parameter is
+    # a request object. Not all languages will generate this method.
     request_object_method: false
+    # Specifies the configuration for retryable codes.
+    # The name must be defined in interfaces.retry_codes_def.
     retry_codes_name: non_idempotent
+    # Specifies the configuration for retry/backoff parameters.
+    # The name must be defined in interfaces.retry_params_def.
     retry_params_name: default
+    # Specifies the default timeout for a non-retrying call.
+    # If the call is retrying, refer to retry_params_name instead.
     timeout_millis: 60000
```

Comment:
Both `flattening` and `required_fields` are not in the template. Can you add commented config section and ask user to uncomment after specifying with the correct value?